frontend/src/test/scala/bloop/testing/BaseSuite.scala

```diff
@@ -210,8 +211,14 @@ abstract class BaseSuite extends TestSuite with BloopHelpers {
       a: AbsolutePath,
       b: AbsolutePath
   )(implicit filename: sourcecode.File, line: sourcecode.Line): Unit = {
-    val filesA = takeDirectorySnapshot(a)
-    val filesB = takeDirectorySnapshot(b)
+    def roundUpTime(s: Seq[AttributedPath]) =
+      s.map { path =>
+        val time = path.lastModifiedTime.toMillis()
+        path.copy(lastModifiedTime = FileTime.fromMillis(time))
+      }
```

Comment:
This was doing my head in for a long time.

For some reason on JDK 17 you get higher resolution timestamps _sometimes_. Therefore string comparison failed:

```
filesA: List[bloop.io.Paths.AttributedPath] = List(AttributedPath(/Foo.class,2022-07-01T12:59:08.443504Z,476))
filesB: List[bloop.io.Paths.AttributedPath] = List(AttributedPath(/Foo.class,2022-07-01T12:59:08.44350436Z,476))
```

This change basically rebuilds the FileTime instance to have milliseconds-only precision.